packages/fiori/src/SideNavigationSelectableItemBase.ts

```diff
@@ -194,13 +201,31 @@ class SideNavigationSelectableItemBase extends SideNavigationItemBase {
 	}
 
 	_onkeydown(e: KeyboardEvent) {
-		if (isSpace(e)) {
+		const isRTL = this.effectiveDir === "rtl";
+
+		if (isSpace(e) || isRight(e) || isLeft(e)) {
 			e.preventDefault();
 		}
 
 		if (isEnter(e)) {
 			this._activate(e);
 		}
+
+		if (isMinus(e) && !this.sideNavCollapsed) {
+			this._activate(e);
+		}
+
+		if (isPlus(e) && this.sideNavCollapsed) {
+			this._activate(e);
+		}
+
+		if ((isRTL ? isLeft(e) : isRight(e)) && this.sideNavCollapsed) {
```

Comment:
`if ((isRTL ? isLeft(e) : isRight(e)) && this.sideNavCollapsed && isInstanceOfSideNavigationItem(this) && this.items.length) {`

You will need to import

`import { isInstanceOfSideNavigationItem } from "./SideNavigationItem.js";`